src/_derive/mod.rs

```diff
@@ -245,6 +245,7 @@
 //! - `rename_all = <string_literal>`: Override default field / variant name case conversion for [`PossibleValue::new`][crate::builder::PossibleValue]
 //!   - When not present: `"kebab-case"`
 //!   - Available values: `"camelCase"`, `"kebab-case"`, `"PascalCase"`, `"SCREAMING_SNAKE_CASE"`, `"snake_case"`, `"lower"`, `"UPPER"`, `"verbatim"`
+//! - `skip`: Ignore this variant
```

Comment:
This is among the attributes for the `enum`, could you move this down to the following section which is for the variants?

Reply:
Oh, my bad. This are the top-level attributes indeed 🤦

Will move it down into the right section, once I'm back home.